prowler/providers/azure/exceptions/exceptions.py

```diff
@@ -9,67 +9,67 @@ class AzureBaseException(ProwlerException):
             "remediation": "Check the Azure environment variables and ensure they are properly set.",
             "file": "{file}",
-            "provider": "Azure",
+            "provider": "{provider}",
         },
         (1915, "AzureNoSubscriptionsError"): {
             "message": "No Azure subscriptions found",
             "remediation": "Check the Azure subscriptions and ensure they are properly set up.",
             "file": "{file}",
-            "provider": "Azure",
+            "provider": "{provider}",
         },
         (1916, "AzureSetUpIdentityError"): {
             "message": "Azure identity setup error related with credentials",
             "remediation": "Check credentials and ensure they are properly set up for Azure and the identity provider.",
             "file": "{file}",
-            "provider": "Azure",
+            "provider": "{provider}",
         },
         (1917, "AzureNoAuthenticationMethodError"): {
             "message": "No Azure authentication method found",
             "remediation": "Check that any authentication method is properly set up for Azure.",
             "file": "{file}",
-            "provider": "Azure",
+            "provider": "{provider}",
         },
         (1918, "AzureBrowserAuthNoTenantIDError"): {
             "message": "Azure browser authentication error: no tenant ID found",
             "remediation": "To use browser authentication, ensure the tenant ID is properly set.",
             "file": "{file}",
-            "provider": "Azure",
+            "provider": "{provider}",
         },
         (1919, "AzureTenantIDNoBrowserAuthError"): {
             "message": "Azure tenant ID error: browser authentication not found",
             "remediation": "To use browser authentication, both the tenant ID and browser authentication must be properly set.",
             "file": "{file}",
-            "provider": "Azure",
+            "provider": "{provider}",
         },
         (1920, "AzureArgumentTypeValidationError"): {
             "message": "Azure argument type validation error",
             "remediation": "Check the provided argument types specific to Azure and ensure they meet the required format.",
             "file": "{file}",
-            "provider": "Azure",
+            "provider": "{provider}",
         },
         (1921, "AzureSetUpRegionConfigError"): {
             "message": "Azure region configuration setup error",
             "remediation": "Check the Azure region configuration and ensure it is properly set up.",
             "file": "{file}",
-            "provider": "Azure",
+            "provider": "{provider}",
         },
         (1922, "AzureDefaultAzureCredentialError"): {
             "message": "Error in DefaultAzureCredential",
             "remediation": "Check that all the attributes are properly set up for the DefaultAzureCredential.",
             "file": "{file}",
-            "provider": "Azure",
+            "provider": "{provider}",
```

Comment:
Can we remove this part since it is always the same?

Reply:
Done